CHANGELOG.md

```diff
@@ -1,9 +1,135 @@
-# Release v1.8-vpp19.01 (2018-12-12)
+# Released versions
+
+- [v2.0.0](#v200)
+- [v1.8.0](#v180)
+- [v1.7.0](#v170)
+- [v1.6.0](#v160)
+- [v1.5.2](#v152)
+- [v1.5.1](#v151)
+- [v1.5.0](#v150)
+- [v1.4.1](#v141)
+- [v1.4.0](#v140)
+- [v1.3.0](#v130)
+- [v1.2.0](#v120)
+- [v1.1.0](#v110)
+- [v1.0.8](#v108)
+- [v1.0.7](#v107)
+- [v1.0.6](#v106)
+- [v1.0.5](#v105)
+- [v1.0.4](#v104)
+- [v1.0.3](#v103)
+- [v1.0.2](#v102)
+
+<!---
+Keep the following order and naming for sections:
+- Compatibility
+- Major Topics
+- Breaking changes
+- New Features
+- Improvements
+- Examples
+- Fixed Bugs
+- Docker Images
+- Documentation
+- Known Issues
+-->
+
+# <a name="v200">(Not yet released) 2.0-vppX.X (2019-X-X)</a>
```

Comment:
We will drop the `vppXX` suffix for tags, because of the VPP multi-version support.

Reply:
true, we will mention all compatible vpps in the compatibility section